packages/amplify_datastore_plugin_interface/lib/src/publicTypes.dart

```diff
@@ -1,8 +1,20 @@
 export 'types/exception/DataStoreException.dart';
 export 'types/exception/DataStoreExceptionMessages.dart';
+export 'types/models/auth_rule.dart';
+export 'types/models/model.dart';
+export 'types/models/model_field.dart';
+export 'types/models/model_field_definition.dart';
+export 'types/models/model_field_type.dart';
+export 'types/models/model_provider.dart';
+export 'types/models/model_schema.dart';
+export 'types/models/model_schema_definition.dart';
+export 'types/models/subscription_event.dart';
```

Comment:
callout: `SubscriptionEvent` was the type that started this conversation, but when I saw these I thought to publicly export all. LMK if makes sense to limit this list a little more.

Reply:
ok, now just limiting to this one.